project_task_personal_stage_auto_fold/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module adds the behavior to change the Personal Stage of the task to the first
+Closed or Folded (Closed Personal Stages has high priority than Folded) stage when
+the task reachs an Stage marked as Closed Stage.
```

Comment:
If you apply my above suggestion, then add this please:

```suggestion
This module adds the behavior to change the Personal Stage of the task to the first
Closed or Folded (Closed Personal Stages has high priority than Folded) stage when
the task reachs an Stage marked as Closed Stage.

If there's a folded personal stage with the same name as the folded shared stage,
that one will be used.

```

Reply:
Done